content/articles/creating-movies-and-animations-using-matlab/index.md

```diff
@@ -0,0 +1,239 @@
+
+6. Saving the movie.
+
+### Matlab functions that can be important at each of the steps
+### 1. Run a simulation or generate a data
+Maybe you have a fancy flight simulator that will run a scenario, kick out all this data, and save it to a file. So all you need here is to load the data. 
+
+It means that You use the `load` function here. Also, if you are familiar with Simulink, you know that you can run a Simulink model from a Matlab script to generate data using the `sim` function.
+
+### 2. Drawing/ rendering the scenario at some point t_x
+This deals with plotting or drawing one single frame of the animation. It means that you will use the plot functions such as `plot`, `plot3`, `surf`. 
+
+A couple of things that can be helpful include the` hold on` function for complicated scenarios or animations. Also, it helps to draw many figures on the same plot.
+
+Because of jamming the `plot` function inside the `for` loop, you have to periodically wipe the slate clean after every time. In this, we use the `clf` function.
+
+### 3. Take a snapshot of the scenario
+Once you have drawn one page of the flipbook, we want to grab that frame and save it into the large flipbook. Matlab has the function `get frame` for doing this.
+
+### 4. Advance time t*k to t*(k+1)
+Like we said before, If we put the process inside a `for` loop or a `while` loop, the step is automatically handled. If you have data that you are simulating that is very temporally timely spaced, you might have tons of data. 
+
+You may not want to plot every single point of your data since that will be a very dense movie. So we will implement the logic of skipping some data and using the `continue` function for this.
+
+### 5. Saving the movie
+To save the movie, we will use `VideoWriter` and `WriteVideo` functions.
+
+### Example
+We want to animate the trajectory of a point. Let us say we run some simulations or have some equations that generate the trajectory at some point in the space. 
+
+The position vector describing where the point is located at any given `x`, `y`,`z` location given time `t` is:
+
+$$
+r(t)=\begin{cases} x(t)=5cos(t)\\y(t)=2sin(t)\\z(t)=t\end{cases}
+$$
+
+The range of `t` is from 0 to `2pi`. If you look at that position vector long enough, you will see that it describes the particle moving in the upward elliptical single helix.
+
+### Implementation in Matlab
+We create a script file for this and do the normal clearance of the workspace and the command window.
+
+```matlab
+%Illustration of animation using Matlab
+clear
+clc
+clear all
+```
+
+Now let us work through the five steps. As we said earlier, the time is going to go through from 0 to `2*pi` and we will use 100 points for this. 
+
+To generate 100 equally spaced points, we use the `linspace` function. We also define our `x`, `y`, and `z` positions.
+
+```matlab
+%% step 1: generate plot
```

Comment:
#### Step 1 - Generate a plot

Could you make the implementation in matlab as a series of steps so the reader can follow through easily. Make the steps as subheadings of the heading `Implementation in Matlab`.